chromium_src/components/permissions/android/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
+  "+brave/browser/widevine",
```

Comment:
That won't solve the problem. The root cause is because we need to set `kAskWidevineInstall` on button click, and any preference r/w needs `chrome/browser`. Desktop can do this because `ui/views/permissions` is in `chrome/browser` but on Android it is part of component.
Cannot do it from Java either. Unless we save a separate preference in Java other than `kAskWidevineInstall`.
Maybe @bridiver will have some better ideas?

Reply:
IMO, having different prefs is fine if it's the only way to avoid this layer violation.

Reply:
@simonhong The layer violation is now solved with a different method (moving `dont_ask_again_` to `PermissionRequest`), see comment by @goodov below.